Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml

```
@@ -27,10 +27,10 @@
       path: /Audio/Weapons/Guns/Gunshots/sniper.ogg
   - type: BallisticAmmoProvider
     capacity: 10
-    proto: CartridgeLightRifle
+    proto: CartridgeHeavyRifle #Sunrise-Edit 7,62x54R
     whitelist:
       tags:
-      - CartridgeLightRifle
+      - CartridgeHeavyRifle #Sunrise-Edit
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)